lib/taskRunner.ts

```diff
@@ -7,12 +7,12 @@ import {Runner} from './runner';
 import {TaskLogger} from './taskLogger';
 
 export interface RunResults {
-  taskId: number;
-  specs: Array<string>;
-  capabilities: any;
+  taskId?: number;
+  specs?: Array<string>;
+  capabilities?: any;
   failedCount: number;
-  exitCode: number;
-  specResults: Array<any>;
+  exitCode?: number;
+  specResults?: Array<any>;
```

Comment:
I knew it. :smile:  (A comment from a previous PR that these should be optional)

Reply:
I also ran into this when working through my PRs.